site/mkdocs.yml

```diff
@@ -29,7 +29,10 @@ nav:
 - Reference:
   - reference/crds.md
   - Connectors API: reference/connectors.md
-  - Components: 
+  - Connectors API (OpenAPI): 
+    - DataCatalog: reference/openapigenerator-doc-datacatalog/README.md
+    - PolicyManager: reference/openapigenerator-doc-policymanager/README.md
```

Comment:
The files linked within are not listed in mkdocs.yml so we get warnings on build. Can you please check what can be done? Either from the generator side (e.g., generate just one page) or from mkdocs side (like auto nav, telling it the pages are ok to be missing, or whatever is possible)

Reply:
Adding a quiet mode flag for now. Please check. Thanks.

Reply:
I don't think that globally running it in quiet mode is good enough. It will actually make it harder to spot mistakes in other parts of the site.